### PR TITLE
bpo-41066: Update the comparison section for  os vs pathlib

### PR DESCRIPTION
**Pull request by srinivasreddy**
https://bugs.python.org/issue41066

### CODE REVIEW

**Comment by srinivasreddy** [comment]
Should be back ported to 3.8, 3.9 as well.

**Review by eamanu — APPROVED** [review]

**Review by nanjekyejoannah — COMMENTED** [review]
A ping with @pitrou for his thought on this.

Looks trivial for a news entry.

**Review by kk456852 — APPROVED** [review]

**Comment by srinivasreddy** [comment]
Agree that it is trivial to have a NEWS entry, so I removed it.

**Review by pitrou — APPROVED** [review]
+1, thank you

File: Doc/library/pathlib.rst [hunk 1]
```diff
@@ -1196,9 +1196,11 @@ os and os.path                         pathlib
 :func:`os.path.exists`                 :meth:`Path.exists`
 :func:`os.path.expanduser`             :meth:`Path.expanduser` and
                                        :meth:`Path.home`
+:func:`os.listdir`                     :meth:`Path.iterdir`
 :func:`os.path.isdir`                  :meth:`Path.is_dir`
 :func:`os.path.isfile`                 :meth:`Path.is_file`
 :func:`os.path.islink`                 :meth:`Path.is_symlink`
+:func:`os.link`                        :meth:`Path.link_to`
```
Comment (pitrou) [inline Doc/library/pathlib.rst:1203]: Do you want to quickly add `os.symlink` and `Path.symlink_to`?

Comment (srinivasreddy) [reply Doc/library/pathlib.rst:1203]: Done.

**Pull request merged**